src/lists/list-item.jsx

```diff
@@ -407,6 +407,7 @@ const ListItem = React.createClass({
     const {
       innerDivStyle,
       style,
+      ...other,
```

Comment:
Can we use the `other` variable from the `render` method? This one has too much variables.